lib/mvi/responses/id_parser.rb

```diff
@@ -5,35 +5,42 @@ module Responses
     class IdParser
       CORRELATION_ROOT_ID = '2.16.840.1.113883.4.349'
       EDIPI_ROOT_ID = '2.16.840.1.113883.3.42.10001.100001.12'
-      ICN_REGEX = /^\w+\^NI\^\w+\^\w+\^\w+$/.freeze
+      ICN_REGEX = /^\w+\^NI\^200M\^USVHA\^P$/.freeze
```

Comment:
This was matching any ID that had an ID Type of 'NI'.  
In every response I've seen from MVI, the first ID is the ICN so this might have never been a practical problem.

From communication with an MVI engineer:
> Will all ICNs have an ID Type of 'NI', assigning authority of '200M' and assigning facility of 'USVHA'?
> > Yes